package.json

```diff
@@ -40,7 +40,7 @@
   "dependencies": {
     "custom-event-polyfill": "joscha/custom-event-polyfill#de24bd9",
     "debounce": "^1.0.0",
-    "weakmap": "0.0.6"
+    "webcomponents.js": "skatejs/webcomponentsjs#skate-v1"
```

Comment:
We can but this is the current one that we're using until the rest of my 3 (out of 4) prs are merged into the webcomponentsjs v1 branch.